src/main/java/com/teamone/clivet/controller/AppointmentRestController.java

```diff
@@ -9,26 +9,34 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("/api/pets")
+@RequestMapping("/api")
 @RequiredArgsConstructor
 public class AppointmentRestController {
 
     private final AppointmentService appointmentService;
 
 
-    @PostMapping("/{petId}/appointments")
+    @PostMapping("/pets/{petId}/appointments")
     public ResponseEntity<?> saveAppointment(@RequestBody AppointmentDto dto,
                                      @PathVariable("petId") Long petId) {
 
         return new ResponseEntity<>(appointmentService.save(dto,petId), HttpStatus.CREATED);
     }
 
-    @GetMapping("/{petId}/appointments")
+    @GetMapping("/pets/{petId}/appointments")
     public ResponseEntity<?> getAppointments (@PathVariable("petId") Long petId){
 
         return new ResponseEntity<>(appointmentService.getByPetId(petId), HttpStatus.OK);
     }
 
-
+    @PutMapping("/appointments/{appId}")
+    public ResponseEntity<?> updateAppointments (@PathVariable("appId") Long appId,
+                                                 @RequestBody AppointmentDto dto){
+        return new ResponseEntity<>(appointmentService.update(appId, dto), HttpStatus.OK);
+    }
+    @DeleteMapping("/appointments/{appId}")
```

Comment:
nie do końca wiem czy to nie będzie miało dalszych konsekwencji na froncie np przy listowaniu (czy ppotem będzie to dostępne)